gluefactory/datasets/oxford_paris_mini_1view_jpldd.py

```diff
@@ -162,6 +177,7 @@ def __init__(self, conf, image_sub_paths: list[str], split):
 
         self.img_dir = DATA_PATH / conf.data_dir
         self.dlsd_kp_gt_folder = self.img_dir.parent / "deeplsd_kp_gt"
+        self.dlsd_line_gt_folder = self.img_dir.parent / "deeplsd_line_gt"
```

Comment:
What has this line to do with Augmentations? I think it was added accidentally?

Reply:
yes this is unrelated to augmentations and is intended for the binary heatmaps